core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/ClusterBootstrapper.java

```diff
@@ -467,4 +474,30 @@ private void validatePollingResultForCancellation(PollingResult pollingResult, S
             throw new CancellationException(cancelledMessage);
         }
     }
+
+    public void validateRotateSaltPassword(Stack stack) {
+        if (!stack.isAvailable()) {
+            throw new BadRequestException("Rotating salt password is only available for stacks in available status");
+        }
+        if (!isChangeSaltuserPasswordSupported(stack)) {
+            throw new BadRequestException("Rotating salt password is not supported with your image version, " +
+                    "please upgrade to an image with salt-bootstrap version >= 0.13.6 (you can find this information in the image catalog)");
```

Comment:
the version could be from the constant so it's in sync if we change the requirement later